mlir/include/mlir/Reducer/Tester.h

```diff
@@ -46,6 +49,9 @@ class Tester {
   /// Return whether the file in the given path is interesting.
   Interestingness isInteresting(StringRef testCase) const;
 
+  void setTestScript(StringRef script) { testScript = script; }
```

Comment:
Why is this needed? I'd not have expected this to change post instantiation. Is this just as the variables aren't available when constructing the tester now as class member?